### PR TITLE
Add Zipkin UI to webapp

### PR DESCRIPTION
**Pull request by kezhenxu94**
- [x] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

image

image

### CODE REVIEW

File: dist-material/release-docs/LICENSE [hunk 1]
```diff
@@ -340,6 +340,7 @@ The text of each license is the standard Apache 2.0 license.
     https://mvnrepository.com/artifact/io.vavr/vavr/0.10.3 Apache-2.0
     https://mvnrepository.com/artifact/io.vavr/vavr-match/0.10.3 Apache-2.0
     https://mvnrepository.com/artifact/io.zipkin.zipkin2/zipkin/2.23.16 Apache-2.0
+    https://mvnrepository.com/artifact/io.zipkin/zipkin-lens/2.23.16 Apache-2.0
```
Comment (wu-sheng) [inline dist-material/release-docs/LICENSE:343]: I think we should resolve lens UI's dependencies, and regenerate the LICENSE file

**Comment by wu-sheng** [comment]
@Fine0830 This would provide webapp integration for Zipkin lens UI. 

Ref https://github.com/apache/skywalking/issues/10000

**Comment by wu-sheng** [comment]
@kezhenxu94 We may consider how to showcase this. How about just opening the Zipkin tracer on Envoy in the mesh? Then we at least could have some spans.

**Comment by kezhenxu94** [comment]
> @kezhenxu94 We may consider how to showcase this. How about just opening the Zipkin tracer on Envoy in the mesh? Then we at least could have some spans.

We already open Envoy SkyWalking tracer in showcase, there is no way to open both Zipkin tracer and SkyWalking tracer at the same time

**Comment by kezhenxu94** [comment]
> > @kezhenxu94 We may consider how to showcase this. How about just opening the Zipkin tracer on Envoy in the mesh? Then we at least could have some spans.
> 
> We already open Envoy SkyWalking tracer in showcase, there is no way to open both Zipkin tracer and SkyWalking tracer at the same time

OK, we didn't open Envoy tracer for now, but I remember @mrproliu planned to do so recently

**Comment by wu-sheng** [comment]
> > @kezhenxu94 We may consider how to showcase this. How about just opening the Zipkin tracer on Envoy in the mesh? Then we at least could have some spans.
> 
> We already open Envoy SkyWalking tracer in showcase, there is no way to open both Zipkin tracer and SkyWalking tracer at the same time

I can't find a trace including the envoy span. I think we don't, otherwise, the topology should be very strange.(the alias name could be strange unless we set the `not-analysis` attribute to true for the segments from envoy)
We just instrument the services, and sidecars forward the `sw8` headers.

Also, we even don't have a trace tab on the mesh dashboard.

File: oap-server-bom/pom.xml [hunk 2]
```diff
@@ -304,6 +304,12 @@
                 <artifactId>zipkin</artifactId>
                 <version>${zipkin.version}</version>
             </dependency>
+            <!-- SkyWalking Web UI -->
+            <dependency>
+                <groupId>io.zipkin</groupId>
+                <artifactId>zipkin-lens</artifactId>
+                <version>${zipkin.version}</version>
```
Comment (wu-sheng) [inline oap-server-bom/pom.xml:311]: We should bump up the version to `2.23.19`?(maybe)

File: dist-material/release-docs/LICENSE (hunk 1 above)
Comment (kezhenxu94) [reply dist-material/release-docs/LICENSE:343]: To regenerate the license file (and make it possible to automatically generate it as what we are doing now), we have to copy the `package.json` and `package-lock.json` from Zipkin Lens project, also Zipkin Lens put all dependencies (including development dependencies) into `dependencies` section so we would also include many un-distributed licenses.

**Comment by wu-sheng** [comment]
> > > @kezhenxu94 We may consider how to showcase this. How about just opening the Zipkin tracer on Envoy in the mesh? Then we at least could have some spans.
> > 
> > 
> > We already open Envoy SkyWalking tracer in showcase, there is no way to open both Zipkin tracer and SkyWalking tracer at the same time
> 
> OK, we didn't open Envoy tracer for now, but I remember @mrproliu planned to do so recently

@mrproliu and you could decide whether we need Zipkin context propagation in our demo service. That is your call.
It is just more fancy if we did.

**Comment by mrproliu** [comment]
> > > @kezhenxu94 We may consider how to showcase this. How about just opening the Zipkin tracer on Envoy in the mesh? Then we at least could have some spans.
> > 
> > 
> > We already open Envoy SkyWalking tracer in showcase, there is no way to open both Zipkin tracer and SkyWalking tracer at the same time
> 
> OK, we didn't open Envoy tracer for now, but I remember @mrproliu planned to do so recently

I just noticed that envoy generates the zipkin format trace context by default when forwarding HTTP protocols. If we want to show that both envoy and skywalking traces are included, we can enhance the demo service to support both skywalking and zipkin traces.

File: dist-material/release-docs/LICENSE (hunk 1 above)
Comment (wu-sheng) [reply dist-material/release-docs/LICENSE:429]: OK, this seems a little crazy :(
It looks like we are over declaring more than we should be.

Comment (wu-sheng) [reply dist-material/release-docs/LICENSE:429]: Let's keep the lib dependencies for now. Copy the package and lock files seem easy to be out-of-date too.

Comment (kezhenxu94) [reply dist-material/release-docs/LICENSE:429]: > Let's keep the lib dependencies for now. Copy the package and lock files seem easy to be out-of-date too.

I don't think we will update the lens dependency too often, this feature is not in our main path and the lens dependency has only static compiled html/css/js files

File: dist-material/release-docs/LICENSE [hunk 3]
```diff
@@ -195,21 +195,104 @@ The text of each license is also included in licenses/LICENSE-[project].txt.
     https://npmjs.com/package/vis-timeline/v/7.5.1 7.5.1 (Apache-2.0 OR MIT)
     https://npmjs.com/package/vis-util/v/5.0.3 5.0.3 (Apache-2.0 OR MIT)
 
+========================================================================
```
Comment (wu-sheng) [inline dist-material/release-docs/LICENSE:198]: It seems you added a new zipkin-License, I think it is good.
Then the following new could be removed?

Comment (wu-sheng) [reply dist-material/release-docs/LICENSE:198]: We could add a line `Zipkin-lens dependences are attached in `zipkin-LICENSE` file. Apache 2.0`(Or something similar)

**Review by wu-sheng — APPROVED** [review]
LGTM

**Pull request merged**